skopt/tests/test_searchcv.py

```diff
@@ -78,6 +83,22 @@ def test_searchcv_runs(surrogate, n_jobs, n_points):
     assert_greater(opt.score(X_test, y_test), 0.9)
 
 
+def test_parallel_cv():
+    """
+    Test whether two parallel jobs acutally work faster than
+    one job.
+    """
+
+    start_time = time.time()
+    test_searchcv_runs(surrogate='gp', n_jobs=1, n_points=1, cv=10)
+    time_one_job = start_time - time.time()
+    start_time = time.time()
+    test_searchcv_runs(surrogate='gp', n_jobs=2, n_points=1, cv=10)
+    time_two_jobs = start_time - time.time()
+
+    assert_greater(time_one_job, time_two_jobs, 'Two jobs are not faster than one job')
```

Comment:
😄

Let's see how often this gives a false alarm, then we might have to think of a better way of detecting two jobs vs one job.

Reply:
I added a guard to skip the test if there is only one CPU. Apparently, it doesn't help much.